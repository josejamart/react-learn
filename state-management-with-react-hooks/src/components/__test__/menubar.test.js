import React from "React";
import Menubar from "../menubar";
import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router'

describe('Menubar -> Component', () => {

    it('Match snapshot', () => {
        const menubar = renderer
            .create(
                <StaticRouter location="someLocation" >
                    <Menubar userName="User name" />
                </StaticRouter>)
            .toJSON();
        expect(menubar).toMatchSnapshot();
    });
})