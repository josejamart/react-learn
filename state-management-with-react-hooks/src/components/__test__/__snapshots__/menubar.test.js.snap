// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menubar -> Component Match snapshot 1`] = `
<nav>
  <div
    className="nav-wrapper"
  >
    <span
      className="band-logo right"
    >
      User name
    </span>
    <ul
      className="left"
    >
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home Page
        </a>
      </li>
      <li>
        <a
          href="/admin"
          onClick={[Function]}
        >
          Admin Page
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
